File: QuestõesPy/questao_1.py
def q1_remover_duplicados(nums):
    nums = list(set(nums))
    return nums

nums = [1,2,2,3,3,4,5,5,5,6,6,6,6,7,8,9,10,10,10]
print (q1_remover_duplicados(nums))

File: QuestõesJs/questao_1.js
function q1_remover_duplicados(nums) {
    nums = [...new Set(nums)];
    return nums;
}

const nums = [1, 2, 2, 3, 3, 4, 5, 5, 5, 6, 6, 6, 6, 7, 8, 9, 10, 10, 10];
console.log(q1_remover_duplicados(nums));

